yugabyte/src/yugabyte/core.clj

```diff
@@ -26,52 +26,69 @@
             [yugabyte.ycql.set]
             [yugabyte.ycql.single-key-acid]
             [yugabyte.ysql.bank]
-            [yugabyte.ysql.counter])
+            [yugabyte.ysql.counter]
+            [yugabyte.ysql.long-fork]
+            [yugabyte.ysql.multi-key-acid]
+            [yugabyte.ysql.set]
+            [yugabyte.ysql.single-key-acid])
   (:import (jepsen.client Client)))
 
-(def noop-test (fn [opts] (merge tests/noop-test opts)))
-
-(def sleep-test
-  (fn [opts] (merge tests/noop-test
-                    {:client (reify Client
-                               (setup! [this test]
-                                 (let [wait-sec 120] (info " === Sleeping for" wait-sec "s ===")
-                                                     (Thread/sleep (* wait-sec 1000))))
-                               (teardown! [this test])
-                               (invoke! [this test op] (assoc op :type :ok))
-                               (open! [this test node] this)
-                               (close! [this test]))}
-                    opts)))
+(defn noop-test
+  "NOOP test, exists to validate setup/teardown phases"
+  [opts]
+  (merge tests/noop-test opts))
 
-(defn with-client
-  [workload client]
-  "Wraps a workload function to add :client entry to the result"
-  (fn [opts] (assoc (workload opts) :client client)))
+(defn sleep-test
+  "NOOP test that gives you time to log into nodes and poke around, trying stuff manually"
+  [opts]
+  (merge tests/noop-test
+         {:client (reify Client
+                    (setup! [this test]
+                      (let [wait-sec 600] (info "Sleeping for" wait-sec "s...")
```

Comment:
the is a `time-limit` command line parameter, can we reuse it here instead of hard-coding 600 seconds?

Reply:
`time-limit` comes from Jepsen itself and it causes test to fail after the given time. Overriding this behaviour would be non-trivial, not sure it's worth it.

Reply:
Actually, I take that back - `time-limit` is already being used in our generators, so we could do  the same here.